org-formation/050-costs/cur.yaml

```diff
@@ -0,0 +1,24 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'Monthly Cost and Usage Report'
+Parameters:
+  Bucket:
+    Type: String
+  Frequency:
+    Type: String
+    Default: "monthly"
+    AllowedValues:
+    - "monthly"
+    - "weekly"
+    - "hourly"
+Resources:
+  MonthlyCostReport:
```

Comment:
I think it might make more sense to just create additional resources here when we want to create a additional reports.  if you agree then maybe we don't need to parameterize `Frequency`?

Reply:
yeah, done